src/Raven.Server/Smuggler/Documents/DatabaseSource.cs

```diff
@@ -464,13 +464,17 @@ public IEnumerable<SubscriptionState> GetSubscriptions()
         public IEnumerable<TimeSeriesItem> GetTimeSeries(List<string> collectionsToExport)
         {
             Debug.Assert(_context != null);
+            var isFull = _startDocumentEtag == 0;
 
             if (collectionsToExport?.Count > 0)
             {
                 foreach (var collection in collectionsToExport)
                 {
                     foreach (var ts in _database.DocumentsStorage.TimeSeriesStorage.GetTimeSeriesFrom(_context, collection, _startDocumentEtag, long.MaxValue))
                     {
+                        if (isFull && ts.Segment.NumberOfLiveEntries == 0)
```

Comment:
Actually... doesn't it affect last etag processed? Shouldn't this be handled in `ProcessTimeSeries` ?